src/tests/ftest/util/dmg_utils.py

```diff
@@ -74,7 +75,61 @@ def storage_scan(self, verbose=False):
         self.set_sub_command("storage")
         self.sub_command_class.set_sub_command("scan")
         self.sub_command_class.sub_command_class.verbose.value = verbose
-        return self._get_result()
+        self.result = self._get_result()
+        self.log.info("--- self.result.stdout ---")
+        self.log.info(self.result.stdout)
```

Comment:
Yes, but it doesn't have the metadata on the left, so it makes it easier to test the regex. It's extra, but job.log for this test has near 600 lines, so the impact is small.

Reply:
We could disable the verbose mode (`DmgCommand.verbose = False`) to prevent the duplicate information, but the few additional lines may not make a big difference.